src/server/test/web/readingsLineMeterRangeRaw.js

```diff
@@ -26,7 +27,70 @@ mocha.describe('readings API', () => {
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.RAW,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.UNIT,
+                            suffix: '',
+                            displayable: Unit.displayableType.ALL,
+                            preferredDisplay: true,
+                            note: 'Celsius'
+                        };
+                        const unitDegrees = {
+                            // u7
+                            name: 'Degrees',
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.RAW,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.METER,
+                            suffix: '',
+                            displayable: Unit.displayableType.NONE,
+                            preferredDisplay: false,
+                            note: 'special unit'
+                        };
+                        const unitDataDegreesC = [unitC, unitDegrees];
+
+                        const conversionDataDegreesC = [
+                            {
+                                // c5
+                                sourceName: 'Degrees',
+                                destinationName: 'C',
+                                bidirectional: false,
+                                slope: 1,
+                                intercept: 0,
+                                note: 'Degrees → C'
+                            }
+                        ];
+
+                        const meterDataDegrees = [
+                            {
+                                name: 'Temp Fahrenheit in Celsius',
+                                unit: 'Degrees',
+                                defaultGraphicUnit: 'C',
+                                displayable: true,
+                                gps: undefined,
+                                note: 'special meter for raw temp data',
+                                file: 'test/web/readingsData/readings_ri_15_days_75.csv',
+                                deleteFile: false,
+                                readingFrequency: '15 minutes',
+                                id: METER_ID
+                            }
+                        ];
+                        await prepareTest(unitDataDegreesC, conversionDataDegreesC, meterDataDegrees);
+                        // Get the graphic unit ID for 'C'
+                        const graphicUnitIdC = await getUnitId('C');
+
+                        const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_line_range_ri_15_mu_kW_gu_kW_st_-inf_et_inf.csv');
+
+                        const res = await chai.request(app)
+                            .get(`/api/unitReadings/line/meters/${METER_ID}?timeInterval=${ETERNITY.toString()}&graphicUnitId=${graphicUnitIdC}`);
```

Comment:
I know this works but could you please use ``.query({ timeInterval: ETERNITY.toString(), graphicUnitId: unitId })`` since this is the way chai does it and it is consistent with the other tests.